uiowa/uiowa
# Callouts

Depends on a review and merge of https://github.com/uiowa/uids/pull/874
Resolves #6254 





# How to test


`git checkout callout-ckeditor && git pull && ddev composer install && ddev blt frontend && ddev blt ds --site=default && ddev drush @default.local uli`
Edit an article and set up some callouts using the Source view of the `filtered_html` body text field
Some simple examples like:
```
<div class="callout inline--size-small inline--align-right">
<div>
<p>Here's a sample callout. It's simple and plain.</p>
</div>
</div>
```
or
```
<div class="callout inline--size-medium inline--align-left">
<div>
<h4 class="headline headline--serif headline--underline">Here's a bigger callout</h4>
<p>It features a headline and a bit more content. It's also larger, and floated left instead of right.</p>
</div>
</div>
```
Play around with https://uids.brand.uiowa.edu/latest/?path=/story/elements-inline--default, and drop various callout creations in and see how they work

Check that you can see the callouts within the WYSIWYG, and that they render appropriately on the page

## Now migration updates
Code review is fine, full work and testing on this can be done in #6241.
